### PR TITLE
Upload testreports of Madagascar release

### PR DESCRIPTION
**Pull request by joseph-v**
**What type of PR is this?**
Upload testreports of Madagascar release

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: testreports/releases/Madagascar/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Test Reports
+
```
Comment (skdwriting) [inline testreports/releases/Madagascar/README.md:2]: We can give the link to the test report. Later find a better way to provide
If you upload binary file for test reports, each fix, you need to reupload. Please check which is better

Comment (joseph-v) [reply testreports/releases/Madagascar/README.md:2]: Thanks, removed binary file and kept link

Comment (skdwriting) [reply testreports/releases/Madagascar/README.md:2]: Given link has SODA_Delfin_Lamu_Test_Report.xlsx - wrong file?!

**Review by skdwriting — APPROVED** [review]
LGTM

**Review by sushanthakumar — APPROVED** [review]

**Review by sushanthakumar — APPROVED** [review]

**Pull request merged**